CHANGELOG.md

```diff
@@ -18,8 +18,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Due to the integration of the Data Catalog into the portal, following changes have been made:
```

Comment:
From another PR:

- [ ] fix the release guide:
  - [ ]  choosing a compatible EDC CE version
  - [ ] setting it in the versions
  - [ ] including the compatible link (release tag-pinned docs link) to the deployment guide of the crawler
  - [ ] working through the deployment notes for the crawler and including them in the deployment migration notes